pkg/controller/subscription/subscription_controller.go

```diff
@@ -136,7 +136,14 @@ func (r *ReconcileSubscription) HandleUpgrades(ctx context.Context, rhmiSubscrip
 
 	if !rhmiConfigs.IsUpgradeServiceAffecting(latestRHMICSV) || canUpgradeNow {
 		eventRecorder := r.mgr.GetEventRecorderFor("RHMI Upgrade")
-		err = rhmiConfigs.ApproveUpgrade(ctx, r.client, latestRHMIInstallPlan, eventRecorder)
+
+		// TODO: investigate a better approach to getting RHMI rather than hardcoding values
+		installation := &integreatlyv1alpha1.RHMI{}
+		err = r.client.Get(ctx, k8sclient.ObjectKey{Name: "rhmi", Namespace: "redhat-rhmi-operator"}, installation)
```

Comment:
Is it possible for a user to install the operator in a different namespace? If it is possible then we would never get to do an upgrade. 

On a side note if it is possible for the user to install the operator in a different namespace, what namespaces would the products get installed in? I am guessing that would stay the same.

Reply:
Good question, I don't think the operator can be installed in another namespace, but would be good to find a way of not having to it hardcoded

https://github.com/integr8ly/integreatly-operator/blob/master/pkg/controller/subscription/subscription_controller.go#L38